.github/ISSUE_TEMPLATE/SUPPORT_QUESTION.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 💬 Support Question
+about: If you have a question, please check out our Slack or StackOverflow!
+
+---
+
+--------------^ Click "Preview" for a nicer view!
```

Comment:
I'd wrap it in a comment so it's invisible in preview

Reply:
Looks great: https://github.com/janpio/cordova-github-templates/issues/new?template=SUPPORT_QUESTION.md